src/lib/eslint/TypescriptEslintStrategy.ts

```diff
@@ -17,11 +17,13 @@ export enum TYPESCRIPT_ENGINE_OPTIONS {
 
 const ES_PLUS_TS_CONFIG = {
 	"parser": "@typescript-eslint/parser",
-	"extends": [
-		"eslint:recommended",
-		"plugin:@typescript-eslint/recommended",
-		"plugin:@typescript-eslint/eslint-recommended"
-	],
+	"baseConfig": {
+		"extends": [
+			"eslint:recommended",
```

Comment:
Maybe I'm misunderstanding, but do we want `eslint:recommended` here? Won't that make this run all of the default ESLint rules in addition to the TS rules?